packages/cascara/src/modules/DataJson/DataJson.js

```diff
@@ -31,25 +32,39 @@ const DataJson = ({
   const { isEditing, formMethods } = useContext(ModuleContext);
   const jsonValue = JSON.stringify(value, null, 4);
 
+  const conditionalLabelProps = getConditionalLabelProps(label, isLabeled);
+
   const renderEditing = (
-    <label htmlFor={label}>
-      {label && isLabeled && <span className={styles.Label}>{label}</span>}
+    <>
+      {label && isLabeled && (
+        <label htmlFor={attribute || label}>
+          <span className={styles.LabelText}>{label || attribute}</span>
+        </label>
+      )}
       <Input
+        {...conditionalLabelProps}
         {...rest}
         as={TextareaAutosize}
         className={styles.Input}
         defaultValue={jsonValue}
-        id={label}
+        id={attribute || label}
         name={attribute || label}
         ref={formMethods?.register}
       />
-    </label>
+    </>
```

Comment:
Label tag to wrap input

Reply:
This module is now wrapped by the new `LabeledModule` 👍